src/DuplicateNum.java

```diff
@@ -0,0 +1,21 @@
+public class DuplicateNum {
+
+	void displayDuplicate(int[] intArr) {
+
+		System.out.print(" Duplicated Number is : ");
+		for (int i = 0; i < intArr.length; i++) {
+			for (int j = i + 1; j < intArr.length; j++) {
+				if (intArr[i] == intArr[j]) {
+					System.out.println(intArr[i]);
+
```

Comment:
Just a suggestion here: we should also handle the scenario where what should happen if there is no duplicate in the array. But no need to change it in the code as your code is as per requirement..

Reply:
@ShrutiDJun20 you are correct !